src/axom/slic/CMakeLists.txt

```diff
@@ -67,6 +67,9 @@ set(slic_depends core)
 
 blt_list_append( TO slic_depends ELEMENTS lumberjack IF ${AXOM_ENABLE_LUMBERJACK} )
 blt_list_append( TO slic_depends ELEMENTS dbghelp IF ${WIN32} )
+if (CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
+  list( APPEND slic_depends execinfo )
```

Comment:
Is `execinfo` always available on `FreeBSD`? Or does it have to be installed?

Reply:
Yes, libexecinfo is in the FreeBSD base system: https://svnweb.freebsd.org/base/head/lib/libexecinfo/